src/graphics/fontman.cpp

```diff
@@ -33,6 +33,9 @@ void FontManager::load(const std::string &path, const std::string &id) {
 
 	std::unique_ptr<Common::ReadStream> font(ResMan.getResource(path));
 
+	if (!font)
+		throw std::runtime_error(fmt::format("Font file not found: {}", path));
```

Comment:
Small hint, now that I see it, you can use the CreateException macro from common/exception.h which has also additional info like the source location where the exception occured

Reply:
Thanks for the hint. I used std::runtime_error for consistency with the other exception in the same file. I will change both exceptions to use the macro.